README.md

```diff
@@ -34,6 +34,16 @@ Make use of the many generators for code, try `ember help generate` for more det
 * `ember test`
 * `ember test --server`
 
+### Generating Documentation
+
+You can use `ember-cli-yuidoc` to generate the documentation in the `docs/` directory:
+
+* `ember ember-cli-yuidoc`
+
+You can also automatically generate, update and serve them at[http://localhost:4200/docs/](http://localhost:4200/docs/):
```

Comment:
I added the space after ad and changed the "them" to "it".

The URL is unfortunately not rendered as a link without the explicit syntax. Also, the other URLs in the Document, generated by `ember-cli` have the markup as well.